app.py

```diff
@@ -127,53 +133,82 @@ def message(event, client):
+                    else:
+                        client.chat_postMessage(channel=channel_id, text=bot_messages.already_titled, thread_ts=message_ts)
+                # bot writes response to question 2 into markdown file and asks question 3 afterwards.
                 elif last_question_asked == 2:
                     client.chat_postMessage(channel=channel_id, text=bot_messages.response_recorded, thread_ts=message_thread_ts)
-                    client.chat_postMessage(channel=channel_id, text=bot_messages.question_three, thread_ts=message_ts)
+                    client.chat_postMessage(channel=channel_id, text=bot_messages.questions[2], thread_ts=message_ts)
 
-                    if text == "None":
-                        file_write("No supplemental image was provided.\n\n")
-                    else:
+                    if not text == "None":
                         # Note: only links of images will work. uploading an image file will not be recognized.
-                        file_write(f"![Supplemental Image]({text})\n\n")
-                # bot records response to question 3.
-                # bot also records the originally scraped message and the timestamp of that scraped message.
+                        file_write(f"![Supplemental Image]({text})\n\n", wiki_title)
+
+                # bot writes response to question 3.
+                # bot also writes the originally scraped message and the timestamp of that scraped message.
                 # the timestamp is used to check whether that message has been scraped already. it is used in the function message_in_documentation().
                 elif last_question_asked == 3:
                     client.chat_postMessage(channel=channel_id, text=bot_messages.response_recorded, thread_ts=message_thread_ts)
                     client.chat_postMessage(channel=channel_id, text=bot_messages.wiki_page_made, thread_ts=message_ts)
                     scraped_message = thread_message_history[0]["text"]
                     if text == "None":
-                        file_write(f"## Original Documentation:\n\n{scraped_message}\n\n## Additional Info:\n\n###### Message Timestamp: {message_thread_ts}\n\n")
+                        file_write(f"## Original Documentation\n\n{scraped_message}\n\n## Additional Info\n\nMessage Timestamp: {message_thread_ts}\n\n", wiki_title)
                     else:
-                        file_write(f"## Original Documentation:\n\n{scraped_message}\n\n## Additional Info:\n\n{text}\n\n###### Message Timestamp: {message_thread_ts}\n\n")
+                        file_write(f"## Original Documentation\n\n{scraped_message}\n\n## Additional Info\n\n{text}\n\nMessage Timestamp: {message_thread_ts}\n\n", wiki_title)
+                    subprocess.call(['git', 'add', f"{wiki_title}.md"], cwd="../docs-slackbot.wiki")
+                    subprocess.call(['git', 'commit', '-am', f"A new wiki named {wiki_title} has been uploaded."], cwd="../docs-slackbot.wiki")

```

Comment:
Nit: commit message should include that the wiki page has been uploaded by the slackbot.

Reply:
will make the small change! i didn't think about that.